### PR TITLE
Group Label Changed Comments in timeline

### PR DESCRIPTION
**Pull request by pta2002**
This targets issue #13268, by grouping label changed comments in an issue's timeline, similar to how GitHub does it. Here's an example:

image

### CODE REVIEW

**Comment by pta2002** [comment]
image

Took the chance to group removing and adding as well!

File: templates/repo/issue/view_content/comments.tmpl [hunk 1]
```diff
@@ -161,15 +161,23 @@
...
-					{{if .Content}}{{$.i18n.Tr "repo.issues.add_label_at" .Label.ForegroundColor .Label.Color (.Label.Name|RenderEmoji) $createdStr | Safe}}{{else}}{{$.i18n.Tr "repo.issues.remove_label_at" .Label.ForegroundColor .Label.Color (.Label.Name|RenderEmoji) $createdStr | Safe}}{{end}}
+					{{if and .AddedLabels (not .RemovedLabels)}}
+					{{$.i18n.Tr (TrN $.i18n.Lang (len .AddedLabels) "repo.issues.add_label" "repo.issues.add_labels") (RenderLabels .AddedLabels) $createdStr | Safe}}
+					{{else}}
+					{{if and (not .AddedLabels) .RemovedLabels}}
+					{{$.i18n.Tr (TrN $.i18n.Lang (len .RemovedLabels) "repo.issues.remove_label" "repo.issues.remove_labels") (RenderLabels .RemovedLabels) $createdStr | Safe}}
+					{{else}}
+					{{$.i18n.Tr "repo.issues.add_remove_labels" (RenderLabels .AddedLabels) (RenderLabels .RemovedLabels) $createdStr | Safe}}
+					{{end}}
+					{{end}}
```
Comment (zeripath) [inline templates/repo/issue/view_content/comments.tmpl]:
```suggestion
					{{if and .AddedLabels (not .RemovedLabels)}}
						{{$.i18n.Tr (TrN $.i18n.Lang (len .AddedLabels) "repo.issues.add_label" "repo.issues.add_labels") (RenderLabels .AddedLabels) $createdStr | Safe}}
					{{else if and (not .AddedLabels) .RemovedLabels}}
						{{$.i18n.Tr (TrN $.i18n.Lang (len .RemovedLabels) "repo.issues.remove_label" "repo.issues.remove_labels") (RenderLabels .RemovedLabels) $createdStr | Safe}}
					{{else}}
						{{$.i18n.Tr "repo.issues.add_remove_labels" (RenderLabels .AddedLabels) (RenderLabels .RemovedLabels) $createdStr | Safe}}
					{{end}}
```

**Review by 6543 — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Review by mrsdizzie — APPROVED** [review]

**Comment by zeripath** [comment]
@pta2002 please update your branch.

(In general it is a better and easier if you allow maintainers to update the branch.)

**Comment by pta2002** [comment]
Done

**Comment by zeripath** [comment]
Unfortunately it needs another update

**Comment by techknowlogick** [comment]
🚀

**Pull request merged**